packages/components/src/components/clipboard/clipboard.jsx

```diff
@@ -19,24 +19,20 @@ const Clipboard = ({
     const isMounted = useIsMounted();
     let timeout_clipboard = null;
 
-    const copyToClipboard = text => {
-        const textField = document.createElement('textarea');
-        textField.innerText = text;
-        document.body.appendChild(textField);
-        textField.select();
-        document.execCommand('copy');
```

Comment:
`document.execCommand('copy');` is deprecated and some browsers won't support it anymore.